opencontrail/files/ifmap.ini

```diff
@@ -0,0 +1,12 @@
+[program:ifmap]
```

Comment:
Hmm.. our opencontrail formula remove's it. This was assumed aka fix that state. If the file not exist, you have to reinstall while recreate missing files (common mistake comes here). I will update condition not to render this template if the destination already exist.